docket_scrap/spiders/docket_spider.py

```diff
@@ -0,0 +1,213 @@
+"""
+Scrapy Spider that crawls RIPUC webpage for commission dockets
+run by command "scrapy crawl docket" on cmd
+"""
+
+from w3lib.html import remove_tags
+import re
+import scrapy
+from urllib.parse import urljoin
+import csv
+
+from ..items import Docket
+
+# from scrapy.linkextractors import LinkExtractor
+# from scrapy.http.request import Request
+
+
+class DocketSpider(scrapy.Spider):
+    # name of the spider
+    name = 'docket'
+
+    # url of the target page
+    start_urls = [
+        'http://www.ripuc.ri.gov/eventsactions/docket.html'
+    ]
+
+    # base url of the website category
+    base_url = 'http://www.ripuc.ri.gov/eventsactions/'
+
+    def parse(self, response):
+        """
+        Handling the initial response of the webpage
+        """
+
+        # Extracting the menu body of the webpage
+        body = response.css("table")[2]
+
+        # Extracting the table of the dockets
+        docket_table = body.css("table")[1]
+
+        # Importing all rows
+        rows = docket_table.xpath('tr')
```

Comment:
these 3 lines can be combined into one by using appropriate css selectors(nth-child among one)